py-rse/git-cmdline.Rmd

```diff
@@ -816,19 +1020,19 @@ and Git will ignore anything that matches any of these lines.
 
 ### Places to create Git repositories {#py-rse-git-cmdline-ex-places}
 
-Along with information about the ENIAC project,
-Frances would also like to keep some notes on the UNIVAC.
+Along with information about the Zipf's law project,
+Amira would also like to keep some notes on the Heap's Law.
 Despite her colleagues' concerns,
```

Comment:
Do we need a glossary item for Heaps Law? (Since we've got one for Zipf's Law.)

Reply:
Probably, yes.  @DamienIrving Is [`gloss.md`](https://github.com/merely-useful/merely-useful.github.io/blob/book/gloss.md) the right place for glossary entries?

Reply:
I've added an entry in `gloss.md`, I was unsure becasue there was no entry there for Zipf's Law, but I'm guessing it's in a PR somewhere...

Reply:
[`links.md`](https://github.com/merely-useful/merely-useful.github.io/blob/book/links.md)